sca_aggregate/create_zap_projects.sql

```diff
@@ -0,0 +1,35 @@
+--- Get dcp_bblnumber and dcp_name from dcp_projectbbls
+DROP TABLE IF EXISTS dcp_projectbbl_tmp;
+CREATE TABLE dcp_projectbbl_tmp AS
```

Comment:
Doesn't need to change, just more of an fyi - if these are truly temp tables and you want them treated as such, you can do `CREATE TEMPORARY TABLE my_table AS`. The table is automatically dropped at the end of the current session. Downside obviously is then you can't query it separately to investigate if something is awry in your final result

Reply:
Ahh yeah. I originally thought this was going to be a tmp table but then thought it might be good to have passed the current session, I can just change the name of table